### PR TITLE
0.2.0

### PR DESCRIPTION
**Pull request by winedarksea**
Latest:
	Added Github Pages documentation
	Changed default for `series_id` so it is no longer required if univariate
	Changed default of `subset` to None.
	Removed `weighted` parameter, now passing weights to .fit() alone is sufficient.
	Fixed a bug where 'One or more series is 90% or more NaN' was printing when it shouldn't
	Fixed (or more accurately, reduced) a bug where multiple initial runs were counting as validation runs.
	Fixed bug where validation subsetting was behaving oddly
	Fixed bug where regressor wasn't being passed to validation.
	Renamed preord_ to future_ regressor.
	Renamed sample datasets.
	Allowed export of result_file as .pickle along with more complete object.
	Added model_interrupt parameter to allow for manually skipping models when enabled.
	Made serious efforts to make the code prettier with pylint, still lots to do, however...
	Improved genetic recombination so optimal models should be reached more quickly
	Improved Point to Probabilistic methods:
		'historic_quantile' more stable quantile-based error ranges
		'inferred normal' Bayesian-inspired method
	Metrics:
		Added Scaled Pinball Loss (SPL)
		Removed upper/lower MAE
	Improved ensembling with new parameter options
		Recursive ensembling (ensemble of ensembles) now enabled
	Validation:
		Added 'seasonal' validation method
	Categorical transformer improved, now tolerant to leaving bounds.
	Added remove_leading_zeroes option for convenience.

	Added a number of new Transformer options
		Multiple new Sklearn-sourced transformers (QuantileTransformer, etc)
		SinTrend
		DifferencedDetrend
		CumSumTransformer
		PctChangeTransformer
		PositiveShift Transformer
		Log
		IntermittentOccurrence
		SeasonalDetrend
		bkfilter and cffilter
		DatepartRegression
	Entirely changed the general transformer to add ~~three~~ four levels of transformation.
	Allowed context_slicer to receive direct integer inputs
	Added new 'Detrend' options to allow more sklearn linear models.

	GLM
		Error where it apparently won't tolerate any zeroes was compensated for.
		Speed improvement.
	RollingRegression
		Added SVM model
		Added option to tune some model parameters to sklearn
		Added new feature construction parameters
		Added RNNs with Keras
	GluonTS:
		fixed the use of context_length, added more options to that param
	Dynamic Factor added uncertainty from Statsmodels Statespace
	VARMAX added uncertainty from Statsmodels Statespace

	New models:
		SeasonalNaive model
		VAR from Statsmodels (faster than VARMAX statespace)
		MotifSimulation
		WindowRegression
		TensorflowSTS
		TFPRegression
		ComponentAnalysis

### CODE REVIEW

**Comment by winedarksea** [comment]
Latest:
	Added Github Pages documentation
	Changed default for `series_id` so it is no longer required if univariate
	Changed default of `subset` to None.
	Removed `weighted` parameter, now passing weights to .fit() alone is sufficient.
	Fixed a bug where 'One or more series is 90% or more NaN' was printing when it shouldn't
	Fixed (or more accurately, reduced) a bug where multiple initial runs were counting as validation runs.
	Fixed bug where validation subsetting was behaving oddly
	Fixed bug where regressor wasn't being passed to validation.
	Renamed preord_ to future_ regressor.
	Renamed sample datasets.
	Allowed export of result_file as .pickle along with more complete object.
	Added model_interrupt parameter to allow for manually skipping models when enabled.
	Made serious efforts to make the code prettier with pylint, still lots to do, however...
	Improved genetic recombination so optimal models should be reached more quickly
	Improved Point to Probabilistic methods:
		'historic_quantile' more stable quantile-based error ranges
		'inferred normal' Bayesian-inspired method
	Metrics:
		Added Scaled Pinball Loss (SPL)
		Removed upper/lower MAE
	Improved ensembling with new parameter options
		Recursive ensembling (ensemble of ensembles) now enabled
	Validation:
		Added 'seasonal' validation method
	Categorical transformer improved, now tolerant to leaving bounds.
	Added remove_leading_zeroes option for convenience.

	Added a number of new Transformer options
		Multiple new Sklearn-sourced transformers (QuantileTransformer, etc)
		SinTrend
		DifferencedDetrend
		CumSumTransformer
		PctChangeTransformer
		PositiveShift Transformer
		Log
		IntermittentOccurrence
		SeasonalDetrend
		bkfilter and cffilter
		DatepartRegression
	Entirely changed the general transformer to add ~~three~~ four levels of transformation.
	Allowed context_slicer to receive direct integer inputs
	Added new 'Detrend' options to allow more sklearn linear models.

	GLM
		Error where it apparently won't tolerate any zeroes was compensated for.
		Speed improvement.
	RollingRegression
		Added SVM model
		Added option to tune some model parameters to sklearn
		Added new feature construction parameters
		Added RNNs with Keras
	GluonTS:
		fixed the use of context_length, added more options to that param
	Dynamic Factor added uncertainty from Statsmodels Statespace
	VARMAX added uncertainty from Statsmodels Statespace

	New models:
		SeasonalNaive model
		VAR from Statsmodels (faster than VARMAX statespace)
		MotifSimulation
		WindowRegression
		TensorflowSTS
		TFPRegression
		ComponentAnalysis
Latest:
	Added Github Pages documentation
	Changed default for `series_id` so it is no longer required if univariate
	Changed default of `subset` to None.
	Removed `weighted` parameter, now passing weights to .fit() alone is sufficient.
	Fixed a bug where 'One or more series is 90% or more NaN' was printing when it shouldn't
	Fixed (or more accurately, reduced) a bug where multiple initial runs were counting as validation runs.
	Fixed bug where validation subsetting was behaving oddly
	Fixed bug where regressor wasn't being passed to validation.
	Renamed preord_ to future_ regressor.
	Renamed sample datasets.
	Allowed export of result_file as .pickle along with more complete object.
	Added model_interrupt parameter to allow for manually skipping models when enabled.
	Made serious efforts to make the code prettier with pylint, still lots to do, however...
	Improved genetic recombination so optimal models should be reached more quickly
	Improved Point to Probabilistic methods:
		'historic_quantile' more stable quantile-based error ranges
		'inferred normal' Bayesian-inspired method
	Metrics:
		Added Scaled Pinball Loss (SPL)
		Removed upper/lower MAE
	Improved ensembling with new parameter options
		Recursive ensembling (ensemble of ensembles) now enabled
	Validation:
		Added 'seasonal' validation method
	Categorical transformer improved, now tolerant to leaving bounds.
	Added remove_leading_zeroes option for convenience.

	Added a number of new Transformer options
		Multiple new Sklearn-sourced transformers (QuantileTransformer, etc)
		SinTrend
		DifferencedDetrend
		CumSumTransformer
		PctChangeTransformer
		PositiveShift Transformer
		Log
		IntermittentOccurrence
		SeasonalDetrend
		bkfilter and cffilter
		DatepartRegression
	Entirely changed the general transformer to add ~~three~~ four levels of transformation.
	Allowed context_slicer to receive direct integer inputs
	Added new 'Detrend' options to allow more sklearn linear models.

	GLM
		Error where it apparently won't tolerate any zeroes was compensated for.
		Speed improvement.
	RollingRegression
		Added SVM model
		Added option to tune some model parameters to sklearn
		Added new feature construction parameters
		Added RNNs with Keras
	GluonTS:
		fixed the use of context_length, added more options to that param
	Dynamic Factor added uncertainty from Statsmodels Statespace
	VARMAX added uncertainty from Statsmodels Statespace

	New models:
		SeasonalNaive model
		VAR from Statsmodels (faster than VARMAX statespace)
		MotifSimulation
		WindowRegression
		TensorflowSTS
		TFPRegression
		ComponentAnalysis
